README.md

```diff
@@ -1,121 +1,33 @@
-# [PROD] RAPIDS.ai Jekyll Site
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/> &nbsp; [RAPIDS.ai](https://rapids.ai.com) Website
 
-## Overview
+Code and content for the [RAPIDS.ai](https://rapids.ai) static public website. This is a refactor from the previous Jekyll backed version to use the [Hugo](https://gohugo.io/about/) static site framework and the [Bulma](https://bulma.io/) css framework. In rare occasions it uses [Apline.js](https://alpinejs.dev/start-here).
 
-This is the PROD repo for all changes for the https://rapids.ai
 
-**All changes** go through this repo and the [deployment process](#deployment-process) outlined below
+## External Content
+Some content is pulled from external sources. Tweets and Blog Posts are generated from the [Data Source Repo](https://rapidsai.github.io/site-data/posts.json), while [Featured Content](https://docs.google.com/spreadsheets/d/e/2PACX-1vQdbJZ13s0QibA4NHcUEdq7ORm7g9fw9oleFWog6rVf90cU4bDiqjEuupdmf38EJw06PpQcb03QNCOI/pub?output=csv), [Events](https://docs.google.com/spreadsheets/d/e/2PACX-1vTUSMyTJWjiYtuV6nvoQIDFSrt6yf00Mt4J9ZaXUMgVi1bAIhcOfnASLhsIH2PhZEbuMCg7Olv9bcZF/pub?output=csv), and [RAPIDS Notices](https://docs.google.com/spreadsheets/d/e/2PACX-1vSPxywR0IGYZsCpqfdaEHNmx4LA9ADoeasCpScd79IhxlI44KjJUYTX52GCxMWgLwN6nEvUXOLDOj31/pub?output=csv) are generated from publicly Published Google Sheets. This may be updated in the future. 
```

Comment:
```suggestion
Some content is pulled from external sources. Tweets and Blog Posts are generated from the [Data Source Repo](https://rapidsai.github.io/site-data/posts.json), while [Featured Content](https://docs.google.com/spreadsheets/d/e/2PACX-1vQdbJZ13s0QibA4NHcUEdq7ORm7g9fw9oleFWog6rVf90cU4bDiqjEuupdmf38EJw06PpQcb03QNCOI/pub?output=csv), [Events](https://docs.google.com/spreadsheets/d/e/2PACX-1vTUSMyTJWjiYtuV6nvoQIDFSrt6yf00Mt4J9ZaXUMgVi1bAIhcOfnASLhsIH2PhZEbuMCg7Olv9bcZF/pub?output=csv), and [RAPIDS Notices](https://docs.google.com/spreadsheets/d/e/2PACX-1vSPxywR0IGYZsCpqfdaEHNmx4LA9ADoeasCpScd79IhxlI44KjJUYTX52GCxMWgLwN6nEvUXOLDOj31/pub?output=csv) are generated from publicly published Google Sheets. This may be updated in the future. 
```

Reply:
actually outdated